GetStream/stream-chat-swift
# Add Split View Support on iPad for Demo App

### 🔗 Issue Link
N/A

### 🎯 Goal
Add Split View support for iPads on the Demo App. This is very important for QAing different scenarios which are not possible with the normal iOS flow. It can also serve as an example of how customers can integrate this in their apps.

### 🛠 Implementation
Adds `UISplitViewController` support in the Demo App.

### 🎨 Changes
![simulator_screenshot_402CC23C-7B6D-4D55-B543-E5C3C8CB8A8F](https://user-images.githubusercontent.com/12814114/147500857-05340e6d-1611-486a-b9a5-d0244f5c0f41.png)

### 🧪 Testing
- Run the App with an iPad
- Should show channels on the Left

- Run the App with an iPhone
- Should not show the split view

### ☑️ Checklist
